please.sh

```diff
@@ -3327,6 +3327,14 @@ create_sdk_artifact () { # [--out=<directory>] [--git-sdk=<directory>] [--archit
 
 		# BusyBox
 		$PREFIX/bin/busybox.exe
+
+		# Asciidoctor (requires Ruby to run)
+		$PREFIX/bin/asciidoctor
+		$PREFIX/bin/asciidoctor.bat
+		$PREFIX/lib/ruby/gems/*/gems/asciidoctor-*/
+		$PREFIX/bin/ruby.exe
+		$PREFIX/bin/ruby*.dll
```

Comment:
Looking at the [package details](https://packages.msys2.org/package/mingw-w64-clang-aarch64-asciidoctor?repo=clangarm64), I _think_ this should do the job. Curious to hear if you have any thoughts or concerns about this.

Reply:
Was able to build `build-installers` locally. Looks like it's still missing a few things:

```
$ asciidoctor --version
`RubyGems' were not loaded.
`error_highlight' was not loaded.
`did_you_mean' was not loaded.
C:/repos/build-installers/clangarm64/bin/asciidoctor:16:in `require': cannot load such file -- rubygems (LoadError)
        from C:/repos/build-installers/clangarm64/bin/asciidoctor:16:in `<main>'
```

Reply:
Looks like I need to add `$PREFIX/lib/ruby/` (the entire folder) instead of just `$PREFIX/lib/ruby/gems/*/gems/asciidoctor-*/`, so I've just force-pushed to make that change. Would that be OK for you?

Reply:
Would you happen to know the difference of the size of the `build-installers` artifact before and after this patch? I am only a bit concerned about making the artifact unnecessarily large.